### ISSUE TITLE
Bug while upgrading from version 1.2.0 to version 1.2.1

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#79]
Originally reported of Google Code with ID 23.

```
<b>What steps will reproduce the problem?</b>
1. create a node 
2. shudown the node
3. modify the node

<b>What is the expected output? What do you see instead?</b>

Command line :

gnt-instance modify -m 256 node

Result :
 
Traceback (most recent call last):
  File "/usr/local/sbin/gnt-instance", line 945, in ?
    override={"tag_type": constants.TAG_INSTANCE}))
  File "/usr/local/lib/python2.4/site-packages/ganeti/cli.py", line 473, in
GenericMain
    result = func(options, args)
  File "/usr/local/sbin/gnt-instance", line 681, in SetInstanceParms
    if opts.hvm_boot_order.lower() == 'default':
AttributeError: 'NoneType' object has no attribute 'lower'

<b>What version of the product are you using? On what operating system?</b>
ganeti-1.2.1 debian-etch

<b>Please provide any additional information below.</b>
In file /usr/local/sbin/gnt-instance, line 680, I've replaced

if opts.hvm_boot_order.lower() == 'default':

by

if opts.hvm_boot_order == None or opts.hvm_boot_order.lower() == 'default':

Is that good or is it a depper problem ?
```
Originally added on 2008-01-26 01:41:45 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi there,

Yep, that is good. It is just missing that check on parsing the command line.
```
Originally added on 2008-01-26 18:35:42 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

This was fixed in version 1.2.2, just released.

Please let us know if it works for you.

thanks!
```
Originally added on 2008-01-30 10:11:37 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release1.2

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
